benchmarking/perception_benchmark_tool/config/benchmark.param.yaml

```diff
@@ -0,0 +1,4 @@
+/**:
+  ros__parameters:
+    prediction_path: "/home/kaan/projects/autoware/src/universe/autoware/benchmarking/perception_benchmark_tool/benchmarking_result/predictions_6.bin"
```

Comment:
Could you fix the path?

Reply:
Hello, Kotaro-san @ktro2828 . So sorry for the late response. I had mandatory conscription last month. 

 I need to re-visit some parts of the code base as soon as possible to make it better, I will re-ping you when everything is ready for review. 

We need to have the lanelet2 map to use the original launch files, I created an issue to make this part optional. It will be useful for other users as well.

Yes, after all the scenes in the dataset are processed, the results are saved under the "perception benchmark_tool/benchmarking result/" location.